fireant/slicer/queries.py

```diff
@@ -124,10 +126,10 @@ def query_data(self, database, table, joins=None,
         :return:
             A pd.DataFrame indexed by the provided dimensions parameters containing columns for each metrics parameter.
         """
-        if rollup and database.query_cls is MySQLQuery:
+        if rollup and issubclass(database.query_cls, (MySQLQuery, RedshiftQuery)):
```

Comment:
I think the comment up mentions postgres as well. so I guess it should be postgres here as well or the comment should get adjusted